File: Tutorial3SuperClassEntities/src/main/java/com/example/tutorial3superclassentities/data/source/model/MeasurementAnglemeter.java
package com.example.tutorial3superclassentities.data.source.model;

import android.arch.persistence.room.ColumnInfo;
import android.arch.persistence.room.Entity;

@Entity(tableName = "table_clinometer_angle")
public class MeasurementAnglemeter extends Measurement {

    @ColumnInfo(name = "angle")
    private double angleSingle = 0;

    public double getAngleSingle() {
        return angleSingle;
    }

    public void setAngleSingle(double angleSingle) {
        this.angleSingle = angleSingle;
    }

}

File: Tutorial3-1SuperClassEntities/src/main/java/com/example/tutorial3superclassentities/data/source/model/MeasurementAnglemeter.kt
package com.example.tutorial3superclassentities.data.source.model

import androidx.room.ColumnInfo
import androidx.room.Entity

@Entity(tableName = "table_clinometer_angle")
data class MeasurementAnglemeter(
    @ColumnInfo(name = "angle")
    var angleSingle: Double = 0.0

) : Measurement()